xlators/mgmt/glusterd/src/glusterd-handler.c

```diff
@@ -1676,7 +1676,6 @@ __glusterd_handle_cli_uuid_get(rpcsvc_request_t *req)
     dict_t *dict = NULL;
     dict_t *rsp_dict = NULL;
     xlator_t *this = THIS;
```

Comment:
It means here (and elsewhere) we probably do not need to unconditionally call 'THIS', since it's used for 'priv' (which you removed) and for error messages. So in the positive path, we should not use 'THIS' at all, only in error flows. Could be done (should be?) in a different patch.

Reply:
In this specific function, it is only used for error messages, which is why I think they can use 'THIS->name' and not unconditionally use 'THIS' even in positive path flows.

Reply:
BTW, a lot of `glusterd` code uses just `gf_msg("glusterd",...)`. It would be helpful to select the only style and consistently use it everywhere.